cumulus_library/study_manifest.py

```diff
@@ -111,19 +111,29 @@ def get_statistics_file_list(self) -> list[str] | None:
         stats_config = self._study_config.get("statistics_config", {})
         return stats_config.get("file_names", [])
 
-    def get_export_table_list(self) -> list[str] | None:
+    def get_export_table_list(self) -> list[tuple] | None:
         """Reads the contents of the export_list array from the manifest
 
-        :returns: An array of tables to export from the manifest, or None if not found.
+        :returns: An array of tuples (table, export type) to export from the manifest,
+        or None if not found.
         """
         export_config = self._study_config.get("export_config", {})
-        export_table_list = export_config.get("export_list", []) or []
-        for table in export_table_list:
-            if not table.startswith(f"{self.get_study_prefix()}__"):
-                raise errors.StudyManifestParsingError(
-                    f"{table} in export list does not start with prefix "
-                    f"{self.get_study_prefix()}__ - check your manifest file."
-                )
+        export_table_list = []
+        for section in [
+            ("export_list", "cube"),
+            ("count_list", "cube"),
+            ("flat_list", "flat"),
+            ("meta_list", "meta"),
```

Comment:
nit: I like the move away from the generic `export_list`, but `count` and `meta` are semantic, while `flat` is syntactic. I know that we maybe don't know the semantic range of flat tables (and/or think the range will be so large, we don't want a field for each), so `flat` is safe. But maybe `count` should be `cube`? And `meta` can continue being a snowflake. Not a strong opinion over here, just bumped on it.

I guess one question is - can we imagine a second kind of cube we'd add in the future? Would we want a new field for it or would we want to put it into the same field?

Reply:
good old naming things problems!

I kind of like semantic for these, in case someone who is not super steeped in this stuff tries to do it - it's a lot clearer to say 'put a count table here from the count generator' than 'make a powerset cube output and put it here'. If I had a semantic name for flat, i would've used it - obviously i don't want to call it 'metric', but maybe there's a better name?

my vibe on cubes is 'maybe at one point we would want a cube that wouldn't be aggregated for some reason'? I'm not sure if I can think of another use case off the dome.

Reply:
Yeah OK - leave it as it is :+1: